client/src/partials/projects/projects.html

```diff
@@ -44,120 +40,188 @@
   </div>
 </div>
 
-<main class="hidden-print">
-  <div class="row">
-
-    <div class="col-xs-6">
-      <div class="panel panel-primary">
-        <div class="panel-heading">
-          {{ 'PROJECT.EXISTING' | translate }}
+<main class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+
+      <div class="col-xs-6">
+        <div class="panel panel-default" style="overflow: auto; max-height: 500px;">
+          <div class="panel-heading">
+            {{ 'PROJECT.EXISTING' | translate }}
+          </div>
+          <table class="table table-condensed table-hover">
+            <thead>
+              <tr>
+                <th>{{ 'COLUMNS.ID' | translate }}</th>
+                <th>{{ 'COLUMNS.NAME' | translate }}</th>
+                <th>{{ 'COLUMNS.ABBR' | translate }}</th>
+                <th>{{ 'COLUMNS.ENTERPRISE_ID' | translate }}</th>
+                <th colspan="2">{{ 'COLUMNS.ACTIONS' | translate }}</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="project in ProjectCtrl.projects | orderBy:'name'" ng-class="{'btn-primary' : editProject.id==project.id }">
+                <td>{{ project.id }}</td>
+                <td>{{ project.name }}</td>
+                <td>{{ project.abbr }}</td>
+                <td>{{ project.enterprise_id }}</td>
+                <td><a ng-click="ProjectCtrl.update(project)" id="project-upd-{{ $index + 1}}"><span class="glyphicon glyphicon-pencil"></span></a></td>
```

Comment:
Both these `<a>` button elements should have `class="btn btn-xs"` to distinguish them from links for our users.